clang/include/clang/Basic/SyncScope.h

```diff
@@ -54,6 +59,16 @@ enum class SyncScope {
 
 inline llvm::StringRef getAsString(SyncScope S) {
```

Comment:
I guess this is a pre-existing problem, but why don't these just match the backend string names?

Reply:
I think it's because this is for AST printing purposes, while the backend strings vary per target.